doc/src/sgml/parallel.sgml

```diff
@@ -329,19 +329,20 @@ EXPLAIN SELECT * FROM pgbench_accounts WHERE filler LIKE '%x%';
 
     <listitem>
       <para> 
-<<<<<<< HEAD
 <!--
-=======
         A prepared statement is executed using a <literal>CREATE TABLE .. AS
         EXECUTE ..</literal> statement.  This construct converts what otherwise
         would have been a read-only operation into a read-write operation,
         making it ineligible for parallel query.
+-->
+準備された文が<literal>CREATE TABLE .. AS EXECUTE ..</literal>文を使って実行される場合。
+この構文を使わないときには読み出し専用の操作だったものが、この構文で読み書き操作に変換され、パラレルクエリには適さないものになります。
```

Comment:
"what otherwise would have been"ですが、検索するとまあまあ使われている慣用句のようです。原文をGoogle翻訳にそのまま食わせると、私の訳よりはるかに自然な日本語になったので、ほぼそのまま使わせてもらうことにしました。
 （Google翻訳では「この構成は、読取り専用操作であったはずのものを読取り /書込み操作に変換し、パラレル問合せには適さないものにします。」